### ISSUE TITLE
Support for Version Catalog / plugins block?

### ISSUE THREAD

**Issue by krzdabrowski** [jaredsburrows/gradle-license-plugin#215]
Currently when I'm trying to add the plugin using Version Catalog entry:
`license  = { id = "com.jaredsburrows.license", version.ref = "license" }`

And using it in `plugins` block in `build.gradle.kts`:
`alias(libs.plugins.license)`

I'm getting this vague error:
```
> Task :app:kaptGenerateStubsDebugKotlin FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:kaptGenerateStubsDebugKotlin'.
> OUTPUT_SIZE
```

Looking at your [last commits](https://github.com/jaredsburrows/gradle-license-plugin/commit/b3b0fc21f79b964892607a28a3d52a72f940ace6) I believe you know what Version Catalog is, same with [`settings.gradle` changes](https://github.com/jaredsburrows/gradle-license-plugin/commit/acc74e397dca35cdb2fd29db277af7ccc2bd0335) introduced in AS Bumblebee that would encourage us to use `plugins` block instead of legacy `classpath` methods.

Could you shed some light on it?

I'm using the newest stable Gradle libraries (Gradle 7.4.2, AGP 7.2.0), error occurs on both `0.9.0` and `0.9.1-SNAPSHOT` versions of your plugin.

**Comment by jaredsburrows** [comment]
I have this working in my example app here: https://github.com/jaredsburrows/android-gif-example/blob/master/gradle/libs.versions.toml#L29.

See this commit: https://github.com/jaredsburrows/android-gif-example/commit/224a6ccf7e5acdf3464c15e09bfb6420475d94b3.

**Comment by krzdabrowski** [comment]
Unfortunately everything looks the same in my code.

This kind of error looks fresh, I see the post on StackOverflow with same error that is week old [here](https://stackoverflow.com/questions/72263977/kotlin-multiplatform-build-output-size-error), maybe it's something Gradle/Version Catalog related?

The weirdest part is that all my other plugins work fine using Version Catalog's syntax but I'm not able to make it work with yours 😟

**Comment by jaredsburrows** [comment]
Is your project open source? Post here or make a repro project?

**Comment by krzdabrowski** [comment]
Unfortunately not, and when I tried to reproduce it in a fresh project, everything works fine so... probably there is an issue related to combination of few plugins/Gradle settings (the only plugin that uses `kapt` in that project is Hilt and currently there's no way to use `ksp` yet). I also checked it's not related to Version Catalog because I got the same error using hardcoded text in `plugins` block.

I will try to keep you updated if I will find more clues related to that issue. For now I'm closing it.
